examples/interactive-containers/interactive-3d-slicer/custom_startup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# 3D Slicer does not properly overwrite output files when they are soft links and not regular files.
+# Since by default FCP sets up soft links from /output to /input, this causes a problem for 3D Slicer. To overcome this, we will delete the soft links in the /output directory and instead copy the relevant files from /input to /output as regular files.
+rm -r /output/*
+# Copy the input directory content to the output directory.
+
+mkdir -p /output/0/dicom_data
+mkdir -p /output/0/files_data
+
+cp -r /input/0/dicom_data/* /output/0/dicom_data/
```

Comment:
What if there are no dicoms in the input dataset? Do we still create /input/0/dicom_data in this case? If not - will this command work?

You may want to structure the script in a way where it first checks if there is a /input/0/dicom_data and if so sets up the appropriate files/links in /output, and then does the same for /input/0/file_data. Extra credit if you have a function that receives either "dicom_data" or "file_data" as input and performs the file copy + linking for that directory as the code is duplicative.

Reply:
In the original behaviour, the directories /input/0/dicom_data and /input/0/file_data are created and the dir tree structure is the same (even though the dirs are empty). 

Modified the script to do that.